src/web/docusaurus/docs/contributing/database-maintenance.md

```diff
@@ -0,0 +1,50 @@
+---
+sidebar_position: 8
+---
+
+# Database Maintenance
+
+The follow describes how to manually work with the Postgres database.
+
+## Backup (Manual)
+
+1. Go to Portainer ([staging](https://dev.portainer.telescope.cdot.systems/) or [production](https://portainer.telescope.cdot.systems/)) and locate the `pg-backup-cron-job` container
```

Comment:
There are several places where we have `[staging](link)` or `[production](link)`, and the links repeat quite a few times. I think we could add the links we need at the bottom of the file and use those throughout:

For example:

```md
[portainer-staging]: https://dev.portainer.telescope.cdot.systems
[portainer-prod]: https://portainer.telescope.cdot.systems
[supabase-staging]: https://dev.supabase.telescope.cdot.systems
[supabase-prod]: https://supabase.telescope.cdot.systems
```

and we can use it like this:

```
1. Go to Portainer ([staging][portainer-staging] or [production][portainer-production])
```